### DISCUSSION TITLE
Prisma try migrate a model that no exist.

### DISCUSSION THREAD

**Discussion by maikelsousa** [prisma/prisma#24736] [Q&A]
### Question

Operating system:   windows 10 64x
Description:

When carrying out the migration, this model 'questions' is informed, which
does not exist and prevents the migration from being carried out.
Already tried:
1- See if it was in other files.
2- Delete the models
3- Change the directory project
4- Reinstall the prism
5- Delete migrations
6- Use the commands npx prisma generate –clean , npx prisma migrate reset
--preview-feature
7- Start the project again
8- Delete and recreate the database
9- Reinstall Postgres
10- Search for a loose file in the root directory of Windows10

I need help to resolve this error.:

p1014
the underlying table for model 'question' does not exist.

### Expected behavior (optional)

![image](https://github.com/prisma/prisma/assets/141464010/b36d493f-2984-43e5-8023-a38263e0c0e5)

### Information about Prisma Schema, Client Queries and Environment (optional)

// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema
```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model Task {
  id         Int       @id @default(autoincrement())
  title      String
  content    String
  finished   Boolean   @default(false)
  categoryId Int?
  category   Category? @relation(fields: [categoryId], references: [id], onDelete: SetNull)
}

model Category {
  id    Int    @id @default(autoincrement())
  name  String
  tasks Task[]
}
```

**Comment by ludralph** [comment]
Hi @maikelsousa 👋 

Can you delete the `_prisma_migrations` table that was created in your database when you initially applied migrations? After deleting it, try to create the migrations again. Do let us know if this resolves the issue for you.

**Comment by maikelsousa** [comment]
yes I tried, and to delete database. not work

**Comment by ludralph** [comment]
Hi @maikelsousa 

The error "P1014: The underlying table for model 'question' does not exist" typically occurs when there's a mismatch between your Prisma schema and the actual database state. However, I noticed that your schema doesn't contain a `question` model.
You need to ensure that your `schema.prisma` file doesn't contain any references to a `question` model. The schema you provided only shows `Task and `Category` models.

If you've tried deleting migrations before, make sure you've also cleared the migration history from your database. You can do this by running:
```
npx prisma migrate reset
```
This command will drop all tables in your database and re-apply all migrations.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
